debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -142,6 +142,36 @@ public static long discontiguousHeaderSize()
 		return J9IndexableObjectDiscontiguous.SIZEOF;
 	}
 
+	/**
+	 * @param objPointer the contiguous array object who's dataAddr field we are accessing
+	 * @throws CorruptDataException 
```

Comment:
should comment mentioned throws NoSuchFieldException as well? - here and next function